bio_market_store/store/templates/user_profile.html

```diff
@@ -34,7 +47,7 @@
           href="#address"
           class="list-group-item list-group-item-action"
           data-bs-toggle="tab"
-          >Address</a
+          >Addresses (Max 2)</a
```

Comment:
Remove (Max 2). Next to the second address, inform a client about additional address "(Optional)".